doc/changes/latest.inc

```diff
@@ -86,6 +86,8 @@ Enhancements
 
 - Reading and writing FIFF files whose filenames end with ``_meg.fif.gz``, ``_eeg.fif(.gz)``, and ``_ieeg.fif(.gz)`` doesn't emit a warning anymore; this improves interobaility with BIDS-formatted datasets (:gh:`8868` by `Richard Höchenberger`_)
 
+- macOS 11 compatibility for `~mne.SourceEstimate` plots (:gh:`8959` by `Richard Höchenberger`_)
```

Comment:
I think the changelog entry should explicitly mention that an env var is being created. Even if right now this only affects MNE, it can potentially affect other programs in future so we should communicate it prominently.

Reply:
My understanding is that these variables will not be visible to any other applications - only to the current Python interpreter and all subprocesses it might spawn.

Reply:
```shell
(base) ➜  ~ echo ${FOO}; python -c "import os; os.environ['FOO'] = 'bar'"; echo ${FOO}


(base) ➜  ~
```

Reply:
Substitute "other programs" with "other Python modules" and the point makes sense though

Reply:
> "other Python modules"

Oh wow, so it's not even global for everything running through the same Python interpreter? Crazy :)

Reply:
No, I meant Dan's point stands -- it will affect those

Reply:
I've improved the change log entry. Feel free to merge if you're happy!